TSE/CCExtensions.cpp

```diff
@@ -2439,55 +2473,63 @@ static PRIMITIVEPROCDEF g_Extensions[] =
 			"iv",	PPFLAG_SIDEEFFECTS,	},
 
 		{	"sysFindObject",				fnSystemFind,	FN_SYS_FIND,	
-			"(sysFindObject source filter) -> list of objects\n\n"
+			"(sysFindObject source criteria) -> list of objects\n\n"
 			
-			"criteria\n\n"
+			"criteria is a string that must specify one or more of the following categories:\n\n"
 			
 			"   b           Include beams\n"
 			"   k           Include markers\n"
 			"   m           Include missiles\n"
 			"   s           Include ships\n"
 			"   t           Include stations (including planets)\n"
-			"   z           Include the player\n"
-			"\n"
+			"   T           Include structure-scale stations\n"
+			"   z           Include the player\n\n"
+			
+			"OR one of the following:\n\n"
+			
+			"   G           Include only stargates\n"
+			"   G:xyz       Include only stargate with ID 'xyz'\n"
+			"   t:xyz       Include only stations with attribute 'xyz'\n"
+			"   T:xyz       Include only structure-scale stations with attribute 'xyz'\n\n"
+			
+			"and may contain any number of the following options:\n\n"
+			
 			"   A           Active objects only (i.e., objects that can attack)\n"
-			//"   B:xyz;      Only objects with attribute 'xyz'\n"
+			//"   B:xyz       Only objects with attribute 'xyz'\n"
 			//"   C           (unused)\n"
-			"   D:xyz;      Only objects with data 'xyz'\n"
-			"   E           Enemy objects only\n"
-			"   F           Friendly objects only\n"
-			"   G           Stargates only\n"
-			"   G:xyz;      Stargate with ID 'xyz'\n"
+			"   D:xyz       Only objects with data 'xyz'\n"
+			"   E           Enemies of the source only\n"
+			"   F           Friends of the source only\n"
 			"   H           Only objects whose base = source\n"
 			"   I:angle     Only objects that intersect line from source\n"
 			"   J           Same sovereign as source\n"
-			"   J:unid;     Sovereign unid = unid\n"
+			"   J:unid      Sovereign unid = unid\n"
```

Comment:
I would keep the semicolons in. Although, strictly they are optional (as you explain below), looking at the source they are almost always used.